client/src/home/CardImage/CardImage.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { trivial_text_maker } from "src/models/text";
+
+import { CardImage } from "./CardImage";
+
+export default {
+  title: "CardImage",
+  component: CardImage,
+};
+
+const Template = (args) => <CardImage {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  img_src: "",
```

Comment:
I think this prop was from the legacy image card components, `CardTopImage` takes an SVG (as a react component) directly. Double check those props.

Can hard code some SVG in here for the story's sake.

Reply:
Added the svg into this component!